web/index.md

```diff
@@ -98,7 +98,7 @@ _"Overview of geoelectrical advances for vineyard study"_
 <!-- <a class="btn btn-primary" href="program.html" role="button">See full program</a> -->
 
 ## Book of abstracts
-Prepare your seminar experience with the [Book of abstracts](https://ilvo_plant-agrogeo24.curve.space/).
+Prepare your seminar experience with the [Book of abstracts](https://agrogeo24.curve.space/).

```

Comment:
On my local git, this is the only change in this file. On github it is also showing changes below, that isn't intentional.